README.md

```diff
@@ -18,6 +18,7 @@ Manual installation takes place in two steps. The code must be installed via `np
 To install the dependencies:
 ```
 npm install
+npm run fetch
```

Comment:
In my understanding, this downloads poky solely for the purpose of the integration tests. Maybe this should not be mentioned as a dependency for the extension?

Reply:
Exacactly! good idea.